.github/workflows/maven.yaml

```diff
@@ -0,0 +1,24 @@
+name: maven
+on:
```

Comment:
nit: might want to include `permissions: {}` to scope them down to harden security (https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs)

Reply:
@embano1 done